test/unit/pr_checker.test.js

```diff
@@ -510,11 +556,63 @@ describe('PRChecker', () => {
         reviews: approvingReviews,
         commits: simpleCommits,
         collaborators
-      });
+      }, argv);
 
       const status = checker.checkCommitsAfterReview();
       assert.deepStrictEqual(status, true);
     });
+
+    it('should log as expected if passed 1 as flag', () => {
+      const { commits, reviews } = moreThanThreeCommitsAfterReview;
+      const expectedLogs = {
+        warn: [
+          [ '4 commits were pushed since the last review, ' +
```

Comment:
Can we just make it

```
Commits pushed since the last review:
- foo: bar
- quo: qux
- bar: foo
...(use `--max-commits ${totalLength}` to see the full list of commits)
```

Reply:
Sure, i think its a good idea.

Reply:
@joyeecheung changed it.